src/main/java/com/pagsestagio/movieapi/service/FilmeService.java

```diff
@@ -279,4 +293,14 @@ public FilmeResultadoRetornaFilmeOuMensagem deletarFilmePeloIdV2(UUID idRequisic
 
         return retornoDeletarFilme;
     }
+
+
+    private String EventoFilmePesquisadotoJson(EventoFilmePesquisado evento) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
[nb]

repara que você vai criar ObjectMapper toda vez que esse método for chamado.